docs/UserGuide.md

```diff
@@ -139,63 +138,138 @@ Exits the program.
+![Exit](images/exit.png)
 
 #### Expected Output(Failure):
 * Message: "Error: Exiting the program failed."
 
-
-
 ## Person Commands
+
 ### Adding a student : `add`
 
 Adds a student to the system.
 
-#### Format: `add n/NAME m/MAJOR y/YEAR e/EMAIL d/DESCRIPTION t/TUTORIALS sm/SOCIALMEDIA nt/NATIONALITY g/GENDER`
+#### Format: `add n/NAME m/MAJOR y/YEAR e/EMAIL d/DESCRIPTION [t/TUTORIALS] [sm/SOCIALMEDIA] nt/NATIONALITY g/GENDER`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A student can include multiple tutorial groups they are interested in. Add multiple tutorials with multiple `t/`.
+Including social media links and tutorial groups are optional.
 </div>
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A student can have any number of social media links (including 0). Add multiple social media links with multiple `sm/`.
+A student can include multiple tutorial groups they are interested in. Add multiple tutorial groups with multiple `t/`.
+</div>
+
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+A student can have any number of social media links. Add multiple social media links with multiple `sm/`.
 </div>
 
 #### Examples:
-* `add n/John Doe m/Computer Science y/2 e/johnd@u.nus.edu d/I’m a Frontend Developer t/06 19 sm/https://www.linkedin.com/in/john-doe-123456789 nt/local g/m`
-* `add n/Betsy Crowe m/Computer Science y/2 e/betsycrowe@u.nus.edu  d/I’m adept at Backend technologies t/05 sm/ nt/foreigner g/f`
+* `add n/John Doe m/Computer Science y/2 e/johnd@u.nus.edu d/I’m a Frontend Developer t/06 t/19 sm/https://www.linkedin.com/in/john-doe-123456789 nt/local g/m`
+* `add n/Betsy Crowe m/Computer Science y/2 e/betsycrowe@u.nus.edu d/I’m adept at Backend technologies t/05 nt/foreigner g/f`
 
 #### Acceptable Values:
-* Name: Full names with alphabetical characters
-* Major: Valid major names at NUS
-* Year: Numeric year level
-* Email: Valid email address ending in “@u.nus.edu”
-* Description: Maximum 150 characters
-* Tutorials: Two digit integers between 01 and 22 inclusive, each seperated by a space
-* Social Media Link: Valid URL format to social media account (optional to include). Multiple URLs can be added by using `sm/` repeatedly.
-* Nationality: Either 'local' or 'foreigner'
-* Gender: A single character, either 'm' or 'f'
+* Name: Full names with alphabetical characters. Maximum 30 characters.
+* Major: Valid major names at NUS. More information listed below.
+* Year: Numeric year level. 
+* Email: Valid email address ending in “@u.nus.edu”. Maximum 20 characters.
+* Description: Maximum 150 characters.
+* Tutorials: Two digit integers between 01 and 22 inclusive. Multiple tutorial slots can be added by using `t/` repeatedly.
+* Social Media Link: Valid URL format to social media account. Multiple URLs can be added by using `sm/` repeatedly.
+* Nationality: Either 'local' or 'foreigner'.
+* Gender: A single character, either 'm' or 'f'.
+
+#### Valid Majors:
+
```

Comment:
Maybe can make a valid major table so it doesn't look like a chunk of words

Reply:
alright, thanks for the suggestion!